src/Components/CalendarIntegration.tsx

```
@@ -0,0 +1,303 @@
+      case 'device':
+        window.open(webcalUrl, '_blank');
+        break;
+      case 'google':
+        // Google Calendar subscription URL
+        calendarUrl = `https://calendar.google.com/calendar/render?cid=${encodeURIComponent(webcalUrl)}`;
+        window.open(calendarUrl, '_blank');
+        break;
+      case 'outlook':
+        // Outlook calendar subscription URL with name and market locale
+        // Extract lang parameter from the actual link URL
+        const urlParams = new URLSearchParams(link.split('?')[1]);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)